src/commands/lm/lm.ts

```diff
@@ -33,7 +33,7 @@ and configures your Git environment with the right credentials.`,
     .command('lm:setup', { hidden: true })
     .description('Configures your site to use Netlify Large Media')
     .option('-s, --skip-install', 'Skip the credentials helper installation check')
-    .option('-f, --force-install', 'Force the credentials helper installation')
```

Comment:
This is another location where we use `--force-install`, and `-f`. In the new update, it is a unified command of `--force`